app/controllers/v0/form1095_bs_controller.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module V0
+  class Form1095BsController < ApplicationController
+    before_action { authorize :form1095, :access? }
+    before_action :get_1095b, only: :download
+    before_action :get_available_forms, only: :available_forms
+
+    def available_forms
+      render json: { available_forms: @available_forms_arr }
```

Comment:
`_arr` is redundant here.
```suggestion
      render json: { available_forms: @available_forms}
```